ci/scripts/install_gcs_testbench.bat

```diff
@@ -0,0 +1,25 @@
+@rem Licensed to the Apache Software Foundation (ASF) under one
+@rem or more contributor license agreements.  See the NOTICE file
+@rem distributed with this work for additional information
+@rem regarding copyright ownership.  The ASF licenses this file
+@rem to you under the Apache License, Version 2.0 (the
+@rem "License"); you may not use this file except in compliance
+@rem with the License.  You may obtain a copy of the License at
+@rem
+@rem   http://www.apache.org/licenses/LICENSE-2.0
+@rem
+@rem Unless required by applicable law or agreed to in writing,
+@rem software distributed under the License is distributed on an
+@rem "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+@rem KIND, either express or implied.  See the License for the
+@rem specific language governing permissions and limitations
+@rem under the License.
+
+@echo on
+
+set TESTBENCH_GCS_VERSION="v0.36.0"
+
+@REM Install GCS testbench %TESTBENCH_GCS_VERSION%
+python -m pip install  ^
+        "https://github.com/googleapis/storage-testbench/archive/%TESTBENCH_GCS_VERSION%.tar.gz" ^
```

Comment:
How about using `GCS_TESTBENCH_...` instead of `TESTBENCH_GCS_...`?
Because we use `gcs_testbench_...` in other locations.

Reply:
Thanks @kou ! Done